assets/js/pages/ExecutionResults/TargetInfoBox.jsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 
 import { TARGET_CLUSTER, TARGET_HOST } from '@lib/model';
 
-import { ClusterInfoBox } from '@components/ClusterDetails';
-import HostInfoBox from '@components/HostDetails/HostInfoBox';
+import { ClusterInfoBox } from '@pages/ClusterDetails';
+import HostInfoBox from '@pages/HostSettingsPage/HostInfoBox';
```

Comment:
I see here we borrow `HostInfoBox` from `HostSettingsPage`; going by our new structure, shouldn't we move `HostInfoBox` into the `commmon/` directory then?

(Though this may not be for this PR)

Reply:
(similar for `ClusterInfoBox`)

Reply:
I think you're right @jamie-suse but I wouldn't follow through that in this PR :+1:

Reply:
I see there also opportunities to refactor some places, but i would not mind to split it up in a follow up pr :)

Reply:
Moved like suggested :+1: